docs/transaction-api.asciidoc

```diff
@@ -142,7 +134,7 @@ curl http://localhost:8200/v1/transactions \
   --data @docs/data/intake-api/generated/transaction/payload.json
 ------------------------------------------------------------
 
-See examples on how a transaction request to the APM Server can look like:
+See how transactions look like:
```

Comment:
This sentence seems a little awkward. Could you just say "Example transaction requests"? I think it's OK to use an incomplete sentence to introduce an example.

Sorry, don't have time to look closely at the context here to see if my edit makes sense.